HHS/Head-Start-TTADP
# CI: Accessibility scan

**Description of change**
Add accessibility scan of all urls to continuous integration pipeline

Accessibility issues were found by the scan and will take some investigation to fix. I created a new issue, https://github.com/HHS/Head-Start-TTADP/issues/159, for this work and to add the `--exit` flag into the axe command. The exit flag will cause the job to fail when an accessibility issue is found.

**How to test**
- See axe accessibility scan run in [recent circle ci job on fork's main](https://app.circleci.com/pipelines/github/adhocteam/Head-Start-TTADP/667/workflows/6a11ff16-119e-4f9b-960e-bc2cd5e25a3c/jobs/2899)

**Issue(s)**
* closes https://github.com/HHS/Head-Start-TTADP/issues/41

**Checklist**

- [n/a] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [x] Documentation updated - **_I simply mentioned this in the README, do you think we need an ADR?_**
    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified
